sw/airborne/modules/gps/gps_i2c.c

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2009  ENAC, Pascal Brisset, Michel Gorraz,Gautier Hattenberger
+ * Copyright (C) 2016 Michael Sierra <sierramichael.a@gmail.com>
```

Comment:
It is usually better to keep the previous authors, even when the original code sucks...